Src/IronPython/Runtime/ModuleOptions.cs

```diff
@@ -66,10 +58,6 @@ public enum ModuleOptions {
         /// </summary>
         Interpret = 0x1000,
         /// <summary>
-        /// String Literals should be parsed as Unicode strings
-        /// </summary>
-        UnicodeLiterals = 0x2000,
-        /// <summary>
```

Comment:
Are there actually removed from the flags in c Python as well? Or are they there and just unused?

Reply:
The CO_FUTURE_* flags for these are still defined but unused. As far as I know ModuleOptions is our own enum so I can't think of a good reason to keep these.